sdk/maintenance/Azure.ResourceManager.Maintenance/tests/Scenario/MaintanenceConfigurationCancellationTests.cs

```diff
@@ -81,7 +81,7 @@ public async Task MaintenanceConfigurationCancelTest()
                     }
                     else
                     {
-                        throw ex;
```

Comment:
Not sure why this wasn't caught before, this warning was added in .NET 5 - https://learn.microsoft.com/en-us/dotnet/core/compatibility/code-analysis/5.0/ca2200-rethrow-to-preserve-stack-details